addPlant.css

```diff
@@ -0,0 +1,117 @@
+*{
```

Comment:
You probably don't need to have two CSS files, it's usually better to have just a single one. You should combine this with your style.css